docs/source/getting-started/preparing-to-install.adoc

```diff
@@ -0,0 +1,61 @@
+include::variables.adoc[]
+
+[[installing-minishift]]
+= Preparing to Install {project}
```

Comment:
@robin-owen `Preparing to Install` does not sound good to me. What about `Preparation to Install Minishift` or `Prerequisites to install Minishift`

Reply:
"Prerequisites" makes more sense to me than "Preparation" does. "Prerequisites" also strikes me as more accurate than the current title.